modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -306,52 +302,51 @@ private boolean _isLocalCacheEnabled() {
 
 	private void _notify(
 		long companyId, String className, BaseModel<?> baseModel,
-		Boolean removePortalCache) {
+		boolean clearCache, boolean removeCache) {
 
 		try (SafeCloseable safeCloseable =
 				CompanyThreadLocal.setWithSafeCloseable(companyId)) {
 
-			_notify(className, baseModel, removePortalCache);
+			_notify(className, baseModel, clearCache, removeCache);
 		}
 	}
 
 	private void _notify(
-		String className, BaseModel<?> baseModel, Boolean removePortalCache) {
+		String className, BaseModel<?> baseModel, boolean clearCache,
+		boolean removeCache) {
```

Comment:
@yuhai @marianoalvarosaiz Here we use 2 boolean parameters for the same if-else clause. Semantically, these 2 boolean parameters can not be both `true`, but can be one `true` + one `false`, or both `false`. However, we can't restrict it based on the current design.

If this change is not necessary for the fix, we can keep the original one; if you insist to make the change, I'd recommend using an `enum` or a selection of constants.

Reply:
@dantewang, I removed the last SF commit. Will trigger ci test again

cc @marianoalvarosaiz